x-pack/plugins/apm/public/components/app/ServiceMap/Popover/Contents.tsx

```diff
@@ -51,14 +85,61 @@ const FlexColumnGroup = (props: {
 const FlexColumnItem = (props: { children: React.ReactNode }) =>
   isIE11 ? <div {...props} /> : <EuiFlexItem {...props} />;
 
+const ANOMALY_DETECTION_TITLE = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverTitle',
+  { defaultMessage: 'Anomaly Detection' }
+);
+
+const ANOMALY_DETECTION_INFO = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverInfo',
+  {
+    defaultMessage:
+      'Display the health of your service by enabling the anomaly detection feature in Machine Learning.'
+  }
+);
+
+const ANOMALY_DETECTION_SCORE_METRIC = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverScoreMetric',
+  { defaultMessage: 'Score (max.)' }
+);
+
+const ANOMALY_DETECTION_LINK = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverLink',
+  { defaultMessage: 'View anomalies' }
+);
+
+const ANOMALY_DETECTION_ENABLE_TEXT = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverEnable',
+  {
+    defaultMessage:
+      'Enable anomaly detection from the Integrations menu in the Service details view.'
+  }
+);
+
 export function Contents({
   selectedNodeData,
   isService,
   label,
   onFocusClick,
   selectedNodeServiceName
 }: ContentsProps) {
-  const frameworkName = selectedNodeData[SERVICE_FRAMEWORK_NAME];
+  // Anomaly Detection
+  const severity = selectedNodeData.severity;
+  const maxScore = selectedNodeData.max_score;
+  const actualValue = selectedNodeData.actual_value;
+  const typicalValue = selectedNodeData.typical_value;
+  const jobId = selectedNodeData.job_id;
```

Comment:
These are all `any` because `selectedNodeData`  is vaguely typed. Currently:

```js
  selectedNodeData: cytoscape.NodeDataDefinition;
```

Should be changed to something like:
```ts
  selectedNodeData: ServiceNode;
```

where `ServiceNode` an intersection type:
```ts
type ServiceNode = cytoscape.NodeDataDefinition & TypeWithAnomalyDetectionFields
```

I expect `TypeWithAnomalyDetectionFields` is defined somewhere on the backend like:
```ts
interface TypeWithAnomalyDetectionFields {
  severity?: string;
  maxScore?: string;
 ...
}